chain/network/src/network_protocol/network.proto

```diff
@@ -373,11 +373,14 @@ message SignedTransaction {
 // Wrapper of borsh-encoded RoutedMessage
 // https://github.com/near/nearcore/blob/1a4edefd0116f7d1e222bc96569367a02fe64199/chain/network-primitives/src/network_protocol/mod.rs#L295
 message RoutedMessage {
+  // Deprecated
+  reserved 3;
+
   bytes borsh = 1;
   // Timestamp of creating the Routed message by its original author.
   google.protobuf.Timestamp created_at = 2;
   // Number of peers this routed message travelled through. Doesn't include the peer that created the message.
-  optional int32 num_hops = 3;
+  uint32 num_hops = 4;
```

Comment:
Just want to confirm, are you completely sure that deprecating this field and reusing the field name is OK?

Reply:
It should be ok:
- reusing the field name is fine as the protobuf encoding is based on the tag numbers
- there could be some data loss with deprecating the old field (for example if a node that uses the old version sends it to one that uses the new one) but the field was not used before anyway

There also is a protobuf backwards compatibility [test](https://github.com/near/nearcore/actions/runs/11292864089/job/31409746916?pr=12188) which passed, but I do not know how it works